Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -172,12 +172,15 @@ def remove_user_from_departing_employee(self, username):
         self._sdk.detectionlists.departing_employee.remove(user_id)
         return user_id
 
-    def get_all_departing_employees(self):
+    def get_all_departing_employees(self, results):
         res = []
         pages = self._sdk.detectionlists.departing_employee.get_all()
         for page in pages:
             employees = page["items"]
-            res.extend(employees)
+            for employee in employees:
```

Comment:
instead of this for loop you can do:
```python
return employees[:results]
```
This will work even if `results` is `None`.

Reply:
ah nevermind, i forgot that this might go over multiple pages